frontend/src/utils/momentUtils.js

```diff
@@ -23,6 +23,22 @@ import _ from 'lodash';
 export const renderRangeWithYear = (start, end) => {
   const panelFormatWithYear = 'DD MMM YYYY';
   const panelFormat = 'DD MMM';
+
+  if(!start){
+    return "invalid startDate";
+  }
+  if(!end){
+    return "invalid endDate";
+  }
+
+  if(start.isAfter(end)){
+    return 'invalid range, endDate must be after startDate';
+  }
+
+  if(start.isSame(end)){
+    return `${start.format(panelFormatWithYear)}`;
```

Comment:
`return start.format(panelFormatWithYear);` is enough in this case.

Reply:
ok done